exporter/templates_test.go

```diff
@@ -52,18 +52,9 @@ func xmlCodeRootNode(codeDisplay models.CodeDisplay) *xml.ElementNode {
 }
 
 func TestReasonTemplate(t *testing.T) {
-	// do not negate reason, r2 compatable
+	// do not negate reason
 	reason := models.CodedConcept{Code: "REASON_CODE_1", CodeSystem: "2.16.840.1.113883.6.1"} // specified in cms9_26.json
 	rootNode := xmlReasonRootNode(reason, false, true)
-	assertXPath(t, rootNode, "//entryRelationship", map[string]string{"typeCode": "RSON"}, nil)
```

Comment:
I removed those test cases because the comment above described them as testing for r2 compatibility. Below, there are tests described as not r2 compatible, so I did not remove that code.